### ISSUE TITLE
change how `appState.scrollX/Y` works

### ISSUE THREAD

**Issue by dwelle** [excalidraw/excalidraw#5965]
In the very early days when we implemented scrolling, for an unknown reason we encoded scroll positions in a way that if you scroll down by 100px, we set `scrollY` as `-100`.

Comes from this commit:

https://github.com/excalidraw/excalidraw/commit/51e19b977eceb8336156e510a5ddde8cef199bb6#diff-0b5adbfe7b36e4ae2f479291e20152e33e940f7f265162d77f40f6bdb5da7405R658-R661

This makes it very unintuitive, especially for public API.

Thoughts on migrating to swap the sign?

/cc @ad1992 @zsviczian

**Comment by ad1992** [comment]
I agree yes we should do it to align with public API

**Comment by zsviczian** [comment]
I agree, scroll x,y is not very intuitive. I ran into this while working on the [presentation](https://youtu.be/HhRHFhWkmCk) script for Obsidian.

I would agree that swapping the sign would make it easier to work with.
